data/map.txt

```diff
@@ -28578,6 +28897,21 @@ planet "Wyvern Station"
 		threshold 3500
 		fleet "Large Militia" 6
 
+planet Xerolophos
+	attributes "requires: gaslining" "gas giant" uninhabited
+	landscape land/nasa10
+	description `Accordingo to astrophysists, Xerolophos appears to be a former dwarf star whose internal reaction was snuffed out at the same time as it lost a significant amount of mass. No known stellar event could have caused this, so it is a subject of much discussion.`
```

Comment:
description `According to astrophysicists, Xerolophos appears to be a former dwarf star. Its internal reaction was snuffed out at the same time as it lost a significant amount of mass. No known stellar event could have caused this, causing it to be a constant subject of much discussion.`

Reply:
Fixed. Thanks!